### DISCUSSION TITLE
the second stage uses an independent neural network

### DISCUSSION THREAD

**Discussion by D19CrMei** [talmolab/sleap#809] [Help!]
in the multi-instance of top-down, the first stage is the anchoring part, and the second stage uses an independent neural network to obtain an image of an anchored instance, and only predict the single anchored instance and peak confidence plot. Convert confidence maps to coordinates using global peak finding. 
I would like to know, what is this independent network? thanks！

**Comment by roomrys** [comment]
Hi @D19CrMei,

In the multi-animal top-down pipeline, two networks are used: first the network for the "Centroid" model and then the network for the "Centered Instance" model. The network for the Centroid model finds a crop of each animal. Then, the network for the Centered Instance model takes as input the crop of the animal and finds the locations of each body part (for each animal/crop). The user can specify which network is used for either model as noted [here](https://github.com/talmolab/sleap/discussions/810?converting=1#discussioncomment-3012372).

Thanks,
Liezl
